package.json

```diff
@@ -44,6 +44,12 @@
                     "prepareCmd": "NODE_OPTIONS=--openssl-legacy-provider sbt -mem 2048 -J-Xmx5120m  \"release release-version ${nextRelease.version} next-version ${nextRelease.version}-SNAPSHOT with-defaults\""
                 }
             ],
+            [
+                "@semantic-release/exec",
+                {
+                "prepareCmd": "yq -i '.appVersion = \"${nextRelease.version}\"' ./infrastructure/charts/mediator/Chart.yaml"
```

Comment:
Didn't you forget to install `yq` in the release workflow yaml?

Reply:
Nice spot